code/game/gamemodes/cult/cult_items.dm

```diff
@@ -334,3 +351,60 @@
 
 	else
 		to_chat(C, "<span class='danger'>The veil cannot be torn here!</span>")
+
+
+/obj/item/weapon/melee/cultblade/ghost
+	name = "eldritch sword"
+	force = 15
+	flags = NODROP
+
+/obj/item/weapon/melee/cultblade/ghost/dropped(mob/living/carbon/human/user)
+	..()
+	qdel(src)
+
+/obj/item/clothing/head/culthood/alt/ghost
+	flags = NODROP
+
+/obj/item/clothing/head/culthood/alt/ghost/dropped(mob/living/carbon/human/user)
+	..()
+	qdel(src)
+
+/obj/item/clothing/suit/cultrobes/alt/ghost
+	flags = NODROP
+
+/obj/item/clothing/suit/cultrobes/alt/ghost/dropped(mob/living/carbon/human/user)
+	..()
+	qdel(src)
+
+/obj/item/clothing/shoes/cult/ghost
+	flags = NODROP
+
+/obj/item/clothing/shoes/cult/ghost/dropped(mob/living/carbon/human/user)
+	..()
+	qdel(src)
+
+//CULT
+
+/obj/item/clothing/suit/space/eva/plasmaman/cultist
+	name = "plasmaman cultist armor"
+	icon_state = "plasmaman_cult"
+	item_state = "plasmaman_cult"
+	desc = "A bulky suit of armour, menacing with red energy. It looks like it would fit a plasmaman."
+	slowdown = 1
+	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
+
+/obj/item/clothing/head/helmet/space/eva/plasmaman/cultist
+	name = "plasmaman cultist helmet"
+	icon_state = "plasmamanCult_helmet0"
+	base_state = "plasmamanCult_helmet"
+	desc = "A helmet designed by the followers of Nar-Sie. It glows menacingly with unearthly flames."

```

Comment:
Is this unavailable for Pyr'Kaus? If so then its fine, otherwise should de-specify, or have it reference a set var like `[station_name()]` does